internal/handlers/pg/msg_delete.go

```diff
@@ -50,152 +50,62 @@ func (h *Handler) MsgDelete(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		return nil, err
 	}
 
-	var deleted int32
-	processQuery := func(i int) error {
-		// get document with filter
-		d, err := common.AssertType[*types.Document](must.NotFail(deletes.Get(i)))
-		if err != nil {
-			return err
-		}
+	sp := new(pgdb.SQLParam)
```

Comment:
I think we agreed to standardise on
```go
var sp pgdb.SQLParam
// ...
foo(&sp)
```

https://dave.cheney.net/practical-go/presentations/gophercon-singapore-2019.html#_use_a_consistent_declaration_style